Sources/MapboxNavigation/TopBannerViewController.swift

```diff
@@ -51,6 +52,10 @@ open class TopBannerViewController: UIViewController {
      */
     public var junctionView: JunctionView = .forAutoLayout(hidden: true)
     
+    private var currentInstruction: VisualInstructionBanner?
```

Comment:
Create the `currentInstruction` to record the latest `VisualInstructionBanner`. When the network is poor, the downloading process is long, we should using the latest instruction to update the `instructionsBannerView` and `nextBannerView`